sdk/communication/AzureCommunication/Tests/CommunicationPolicyTokenCredentialTests.swift

```diff
@@ -0,0 +1,148 @@
+// --------------------------------------------------------------------------
```

Comment:
Anything in the `AzureCommuncation` code should defer to master.

Reply:
This is actually a new test I introduced in the feature branch working with @knvsl so it is not on master.